app/views/manage/request-a-new-code.html

```diff
@@ -0,0 +1,20 @@
+
+{% extends "_templates/form-template.html" %}
```

Comment:
This doesn't need to be a form template. `page-template` would be more appropriate.